Form/Type/ColorSelectorType.php

```diff
@@ -49,6 +49,13 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::YELLOW,
             ),
         ));
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $resolver->setDefaults(array_merge($resolver->getDefinedOptions(), array(
+                'choices' => array_unique(array_flip(Colors::getAll())),
+            )));
+        }
```

Comment:
Is it deprecated, because the keys and values are flipped by default in symfony 3. For symfony 2.8+ we must declare it explicitly to stay BC.